test/test_functional_tensor.py

```diff
@@ -578,6 +579,52 @@ def test_assert_resize_antialias(interpolation):
         F.resize(tensor, size=(5, 5), interpolation=interpolation, antialias=True)
 
 
+@pytest.mark.parametrize('dt', [torch.float32, torch.float64, torch.float16])
+@pytest.mark.parametrize('size', [[10, 7], [10, 42], [42, 7]])
+@pytest.mark.parametrize('interpolation', [BILINEAR, BICUBIC])
+def test_interpolate_antialias_backward(dt, size, interpolation):
+
+    # temporarily hard-code device as CPU, CUDA support will be done later
+    device = "cpu"
+
+    if dt == torch.float16 and device == "cpu":
+        # skip float16 on CPU case
+        return
+
+    torch.manual_seed(12)
```

Comment:
In general, should we set the seed of the tests in a more automated way via pytest?

Reply:
It's fine for now as this is what we're doing almost everywhere else
Once we're finished with pytest porting I'll look into ways to improve the RNG handling in our tests.
One thing I'm wondering is: does `torch.manual_seed(12)` leak the rng for the rest of the tests?

Reply:
> does torch.manual_seed(12) leak the rng for the rest of the tests?

I think yes

Reply:
that's pretty bad :)

but yeah, it's OK for now to use the old pattern.

Reply:
it certainly does leak it and it's bitten us quite a few times in the past.